docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -174,31 +174,15 @@ Follow these steps if you want to use your initial cluster to create and manage
 
 1. Create a workload cluster in one of the following ways:
    
-   * **GitOps**: Recommended for more permanent cluster configurations.
-     1. Clone your git repo and add the new cluster specification. Be sure to follow the directory structure defined on [Manage cluster with GitOps]({{< relref "/docs/tasks/cluster/cluster-flux" >}}):
+   * **GitOps**: See [Manage separate workload clusters with GitOps]({{< relref "/docs/tasks/cluster/cluster-flux#manage-separate-workload-clusters-using-gitops" >}})
```

Comment:
the format of relative links in md file should be [<name>](<relative-path>). And seems like we don't have manage-separate-workload-clusters-using-gitops in our repo.

Reply:
I guess we can either do only relative file link or use full file address with section anchor

Reply:
ok thanks! i will update to use the relative path with the `.md` suffix